includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```
@@ -2007,6 +2020,12 @@ public function process_order_for_confirmed_intent( $order, $intent_id, $save_pa
 
 		if ( ! empty( $error ) ) {
 			WC_Stripe_Logger::error( 'Error when processing payment for order: ' . $order->get_id(), [ 'error_message' => $error->message ] );
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Logger' in method 'process_order_for_confirmed_intent'. (undefined)

(StaticAccess)